x11-wm/emwm/emwm-1.0a.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit optfeature
+
+DESCRIPTION="Enhanced Motif Window Manager (MWM fork). Supports Xinerama, Xrandr, UTF-8/Xft, and relevant Extended Window Manager Hints (EWMH)."
```

Comment:
Too long _but_ you're not to blame! pkgcheck until recently (just released today) had some crazy limit of like, 120-140? It now correctly has 70 IIRC.

Reply:
Maybe EAPI 9 can deprecate `DESCRIPTION` in favour of `SHORT_DESCRIPTION` and the meatadata.xml's 'long description' tag can become 'description' and we can do away with multi-paragraph descriptions altogether.

/s; mostly